lib/Conversion/TorchOnnxToTorch/DefaultDomainQtoZ.cpp

```diff
@@ -863,31 +862,91 @@ void mlir::torch::onnx_c::populateDefaultDomainQtoZ(
 
         Value axesVal;
         if (!binder.tensorOperandAtIndex(axesVal, 1)) {
```

Comment:
The condition here means that either the `axes` arg is present (for version 18 or above it is optional but when present it's available as an input tensor) or if it's not present then either it is absent or it might be present as an attribute for the lower versions of ReduceMax op.